src/components/ContentHighlights/HighlightStepper/HighlightStepperSelectContentDataTable.jsx

```diff
@@ -105,14 +106,14 @@ const BaseHighlightStepperSelectContentDataTable = ({
       isPaginated
       manualPagination
       initialState={{
-        pageSize,
         pageIndex: 0,
+        pageSize,
         selectedRowIds,
       }}
       pageCount={searchResultsPageCount}
       itemCount={searchResultsItemCount}
       initialTableOptions={{
-        getRowId: row => row.aggregationKey,
+        getRowId: row => row?.aggregationKey,
```

Comment:
when would we not have a row?

Reply:
@alex-sheehan-edx We always _should_ have a `row` here. That said, for some reason we ran into the function arg for `getRowId`in the `DataTable` within the "Learner Credit Management" feature was undefined at one point and we needed to do a similar fix like this. Hamzah convinced me to add the `?`, just in case, for this reason. :)